src/main/Duke.java

```diff
@@ -0,0 +1,206 @@
+package main;
```

Comment:
`Duke.java`

- There are several coding standard violations that I did not point out. Please refer to https://se-education.org/guides/conventions/java/index.html.
- Overall, the design for class `Duke` is not very good, e.g. unrelated static methods, SLAP is lacking, readability issues.